.github/workflows/main.yml

```diff
@@ -389,7 +389,7 @@ jobs:
         setup-python: false
 
     - name: Use ccache
-      uses: hendrikmuhs/ccache-action@v1
+      uses: hendrikmuhs/ccache-action@v1.2.3
```

Comment:
Please remove the CI debugging lines.
For debugging it's good to create a sub-branch so changes don't make into main.